src/main/java/io/unityfoundation/auth/HasPermissionRequest.java

```diff
@@ -0,0 +1,13 @@
+package io.unityfoundation.auth;
+
+import io.micronaut.serde.annotation.Serdeable;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+@Serdeable
+public record HasPermissionRequest(
```

Comment:
I think this needs an extra flag indicating that SUBTENANT scope is allowed.  Or is that what' I'm seeing in the AuthController?

Reply:
currently I set subtenant flag as query parameter, but if you think that HasPermissionRequest record would be better position I can change that